ddtrace/tracer/option.go

```diff
@@ -302,8 +305,20 @@ const partialFlushMinSpansDefault = 1000
 func newConfig(opts ...StartOption) *config {
 	c := new(config)
 	c.sampler = NewAllSampler()
+	defaultRate, err := strconv.ParseFloat(getDDorOtelConfig("sampleRate"), 64)
+	if err != nil {
+		log.Warn("ignoring DD_TRACE_SAMPLE_RATE, error: %v", err)
```

Comment:
Hey @darccio we probably should not log this here - this code is running at the init of our customer-facing CLI and is always printed. We cannot remove it because it's coming from one of the transitive dependencies. 

We cannot upgrade due to this. Could you kindly remove this log?

Reply:
Created issue https://github.com/DataDog/dd-trace-go/issues/2749